OpenRA.Game/WorldUtils.cs

```diff
@@ -70,13 +50,15 @@ public static void PlayVoiceForOrders(this World w, Order[] orders)
 				if (o == null)
 					continue;
 
-				if (o.Subject.Destroyed)
+				var orderSubject = o.Subject;
+				if (orderSubject.Destroyed)
 					continue;
 
-				foreach (var v in o.Subject.TraitsImplementing<IOrderVoice>())
-					if (Sound.PlayVoice(v.VoicePhraseForOrder(o.Subject, o),
-						o.Subject, o.Subject.Owner.Country.Race))
-						return;
+				foreach (var voice in orderSubject.TraitsImplementing<IVoiced>())
```

Comment:
This and Selection are still in Game, while ActorExts are in Common. I could move the PlayVoice/HasVoice to Game of course (though I don't know into which file), but I believe the long-term goal was rather to move more stuff from the Game engine to Common, right?